core/src/packets/ip/v6/mod.rs

```diff
@@ -383,7 +395,7 @@ impl Default for Ipv6Header {
             version_to_flow_label: u32::to_be(6 << 28),
             payload_length: 0,
             next_header: 0,
-            hop_limit: 0,
+            hop_limit: DEFAULT_TTL,
```

Comment:
minor: do we have note somewhere that ttl and hop_limit are related in v6 land?

Reply:
no that's more a rfc spec thing. the v4 ttl and v6 hop limit serve the same purpose.